lib/db.ts

```
@@ -1,42 +1,23 @@
-  cached = global.mongoose = { conn: null, promise: null }
+declare global {
+  var prisma: PrismaClient | undefined;
 }
 
-export async function connectToDatabase() {
-    if (cached.conn){
-        return cached.conn;
-    }
-
-    if (!cached.promise) {
+const prisma = globalThis.prisma || new PrismaClient();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)